ibis/backends/postgres/registry.py

```diff
@@ -119,17 +119,12 @@ def _timestamp_truncate(t, expr):
 
 
 def _interval_from_integer(t, expr):
-    arg, unit = expr.op().args
+    arg, _ = expr.op().args
```

Comment:
Since we support node inheritance the  order of `args` isn't that explicit anymore, so I'd prefer using attributed explicitly. Besides that accessing properties should be quicker too.

Reply:
Roger, good catch.